quartz/components/scripts/popover.inline.ts

```diff
@@ -82,8 +82,15 @@ async function mouseEnterHandler(
       const contents = await response.text()
       const html = p.parseFromString(contents, "text/html")
       normalizeRelativeURLs(html, targetUrl)
-      // strip all IDs from elements to prevent duplicates
-      html.querySelectorAll("[id]").forEach((el) => el.removeAttribute("id"))
+      // prepend all IDs inside popovers to prevent duplicates
+      html.querySelectorAll("[id]").forEach((el) => {
+        const targetID = `popover-${el.id}`
+        el.id = targetID
+      })
+      const targetLink = hash.startsWith("#popover")
+        ? hash
+        : `#popover-${hash.slice(1).replace("^", "").replace("%5E", "")}`
```

Comment:
what is `%5E` ? also we should only replace the "^" at the start of the string

Reply:
I'll check if it is still needed, but `%5E` is the HTML escape version of `^`.